### PR TITLE
Adding members of the group to the website

### PR DESCRIPTION
**Pull request by mitchellolsthoorn**
This PR adds all the current members and alumni members of the group to the website

### CODE REVIEW

File: content/authors/wouter-zorgdrager/_index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+# Display name
+title: Wouter Zorgdrager
+
+# Is this the primary user of the site?
+superuser: false
+
+role: Scientific Developer
```
Comment (mitchellolsthoorn) [inline content/authors/wouter-zorgdrager/_index.md:8]: Add " (part-time)" behind the role

File: content/authors/rini-van-solingen/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+# Display name
+title: Rini van Solingen
+
+# Is this the primary user of the site?
+superuser: false
+
+role: Full Professor
```
Comment (mitchellolsthoorn) [inline content/authors/rini-van-solingen/_index.md:8]: Add " (0.2fte)" behind the role

File: content/authors/mauricio-aniche/_index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+# Display name
+title: Mauricio Aniche
+
+# Is this the primary user of the site?
+superuser: false
+
+role: Assistant Professor
```
Comment (mitchellolsthoorn) [inline content/authors/mauricio-aniche/_index.md:8]: Add " (0.2fte)" behind the role

File: content/authors/diomidis-spinellis/_index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+# Display name
+title: Diomidis Spinellis
+
+# Is this the primary user of the site?
+superuser: false
+
+role: Full Professor
```
Comment (mitchellolsthoorn) [inline content/authors/diomidis-spinellis/_index.md:8]: Add " (0.2fte)" behind the role

File: content/authors/dimitri-stallenberg/_index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+---
+# Display name
+title: Dimitri Stallenberg
+
+# Is this the primary user of the site?
+superuser: false
+
+role: Scientific Developer
```
Comment (mitchellolsthoorn) [inline content/authors/dimitri-stallenberg/_index.md:8]: Add " (part-time)" behind the role

**Comment by mitchellolsthoorn** [comment]
For the alumni, can you create author profiles for them too? The affiliation would then be their current affiliation from the table, the role would be the role in the table, the user group would be "Member Alumni". For the period, you can use the following properties:
```
member_start: 2010
member_end: 2019
```

File: content/people/people.md [hunk 6]
```diff
@@ -19,10 +19,12 @@ content:
...
     - Associate Professors
     - Assistant Professors
-    - Researchers
     - Postdoctoral Researchers
+    - Researchers
     - Ph.D. Candidates
     - Scientific Developers
+    - Guests
+    - Support
     - M.Sc. students
```
[the remark this answers is not in the document]
Comment (patrikbartak) [reply content/people/people.md:28]: Done

File: content/authors/rini-van-solingen/_index.md (hunk 2 above)
Comment (patrikbartak) [reply content/authors/rini-van-solingen/_index.md:8]: Done

File: content/authors/mauricio-aniche/_index.md (hunk 3 above)
Comment (patrikbartak) [reply content/authors/mauricio-aniche/_index.md:8]: Done

File: content/authors/diomidis-spinellis/_index.md (hunk 4 above)
Comment (patrikbartak) [reply content/authors/diomidis-spinellis/_index.md:8]: Done

**Pull request merged**